src/org/parosproxy/paros/core/scanner/VariantAbstractQuery.java

```diff
@@ -164,6 +169,7 @@ private String setParameter(HttpMessage msg, NameValuePair originalPair, String
         String encodedValue = (escaped) ? value : getEscapedValue(msg, value);
         NameValuePair pair;
         boolean isAppended;
+        String query = null;

```

Comment:
This could be removed and declared when converting the `sb` to string (would be as before the changes).